### PR TITLE
Automated Test - COMMERCE-6019 | Master

### PR DESCRIPTION
**Pull request by CalumR23**
[COMMERCE-6019](https://issues.liferay.com/browse/COMMERCE-6019)
@Pietro69

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 1]
```diff
@@ -94,13 +94,75 @@ definition {
...
+
+		Type(
+			key_fieldLabel = "Key",
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = "red");
+
+		CommerceEntry.saveEntry();
+
+		Navigator.openURL();
+
```
Comment (CalumR23) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:152]: Here you have to use Navigator to get back to the home page as you cannot exit out of the Commerce Entry once going to it on line 137

**Comment by CalumR23** [comment]
ci:test:sf

**Review by Pietro69 — APPROVED** [review]

**Comment by Pietro69** [comment]
Please add the property test.name.skip.portal.instance = `"CPCommerceProductOptions#EditAndRemoveAnOptionWithValues"`

**Comment by fmarcio** [comment]
New PR sent https://github.com/liferay-commerce/liferay-portal/pull/1165

**Pull request closed without merging**